crates/commands/src/fwd_search.rs

```diff
@@ -65,7 +65,9 @@ impl ForwardSearch {
             return Err(ForwardSearchError::InvalidPath(child.uri.clone()));
         };
 
-        let Some(pdf_path) = parent.path
+        let override_path = workspace.output_filename();
```

Comment:
Here, you can use borrow the path from the config struct instead to avoid cloning both paths.

```rs
let override_path = workspace.config().build.output_filename.as_deref();

let Some(pdf_path) = override_path.or(parent.path.as_deref())
    .and_then(Path::file_stem)
    .and_then(OsStr::to_str)
    .map(|stem| dir.join(format!("{stem}.pdf"))) else 
{
    return Err(ForwardSearchError::InvalidPath(parent.uri.clone()));
};

```